src/frontend/test_runner/tests/testdata/expr.yaml

```diff
@@ -61,7 +61,7 @@
 - sql: |
     /* in-list with misaligned types */
     SELECT true in (3, 1.0, 2);
-  binder_error: 'Bind error: types Boolean and Int32 cannot be matched'
+  binder_error: "Bind error: types Boolean and Int32 cannot be matched"
```

Comment:
Are you using a different version of `yq`?

Reply:
I do not know this repo use `yq` before. After a grep I only found [this](https://github.com/singularity-data/risingwave/blob/606925831c462a99796fccf865adba8a2e91a656/docs/developer-guide.md?plain=1#L326).
Is there any docs about which args should be use with `yq` when formating `expr.yaml` or if there exists some scripts to do this stuff?

Reply:
This file is generated by `./risedev do-apply-planner-test` and the `yq` args should be here:
https://github.com/singularity-data/risingwave/blob/a4d85ede4dd96058668c1fe1a38a4afe5473e879/src/frontend/test_runner/planner_test.toml#L13

Reply:
Thx.

Would it be better to add this line in [the task](https://github.com/singularity-data/risingwave/blob/a4d85ede4dd96058668c1fe1a38a4afe5473e879/src/frontend/test_runner/planner_test.toml#L49)?

If so I gonna file another merge request for this

```sh
    sed -i '1i# This file is generated and formatted by running the task `./risedev do-apply-planner-test`.\n' "$f"
```